### PR TITLE
Update modules.pod6

### PR DESCRIPTION
**Pull request by finanalyst**
Extra clarification for fez versioning, indicate preference for fez, note that if module migrated to fez to remove from p6c

### CODE REVIEW

**Review by dontlaugh — COMMENTED** [review]
This looks good to me.

**Review by dontlaugh — APPROVED** [review]

File: doc/Language/modules.pod6 [hunk 1]
```diff
@@ -1069,8 +1069,20 @@ fez upload
...
+=item To prevent changes to a versioned distribution (remember a distribution may contain more than
+one module), only B<one> upload of a distribution per version is permitted. If a developer makes changes to 
+code in a repository, the changes will not be reflected in the C<fez> system. So changes to code must be
+accompanied by a bump in the version number and the new version uploaded to C<fez>.
+
 =head2 Upload your module to p6c
 
+The Raku community is migrating to the I<fez> ecosystem (see above). For developers
+who have modules on the I<p6c> ecosystem and migrated them to I<fez>, please remove them
+from the I<p6c> ecosystem. If you do not have the commit bit, or ping someone on IRC (#raku at libera.chat) to
```
Comment (Altai-man) [inline doc/Language/modules.pod6:1083]: Looks like a part of the sentence is missing here? Something like "the commit bit, open a pull request for the ecosystem repo, or ping...".

**Review by Altai-man — CHANGES_REQUESTED** [review]

File: doc/Language/modules.pod6 (hunk 1 above)
Comment (dontlaugh) [reply doc/Language/modules.pod6:1083]: Good catch

Comment (coke) [reply doc/Language/modules.pod6:1083]: if you can address this, we can merge this change

Comment (coke) [reply doc/Language/modules.pod6:1083]: @finanalyst - can you fix this item?

**Review by coke — APPROVED** [review]
I'll fix this one item in post.

**Pull request merged**